hack/env.sh

```diff
@@ -10,7 +10,6 @@ F8A_PORT=${BC_SERVER_SERVICE_PORT:-5000}
 POSTGRES_PORT=${PGBOUNCER_SERVICE_PORT:-5432}
 ANITYA_PORT=${ANITYA_SERVICE_PORT:-5000}
 
-export CCS_CELERY_BROKER=amqp://guest@${RABBIT_HOST}:${RABBIT_PORT}
-export CCS_F8A_SERVER=http://${F8A_HOST}:${F8A_PORT}
-export CCS_POSTGRES=postgresql://${POSTGRESQL_USER}:${POSTGRESQL_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRESQL_DATABASE}?sslmode=disable
-export CCS_ANITYA=http://${ANITYA_HOST}:${ANITYA_PORT}
+export F8A_CELERY_BROKER=amqp://guest@${RABBIT_HOST}:${RABBIT_PORT}
+export F8A_POSTGRES=postgresql://${POSTGRESQL_USER}:${POSTGRESQL_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRESQL_DATABASE}?sslmode=disable
+export F8A_ANITYA=http://${ANITYA_HOST}:${ANITYA_PORT}
```

Comment:
Well, these are used in more places, for example in [cucoslib/conf.py](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/cucoslib/conf.py).

I'd be especially careful with `CCS_POSTGRES` which is used on even more places.